tests/Doctrine/Tests/ORM/UnitOfWorkTest.php

```diff
@@ -250,6 +250,8 @@ public function testNoUndefinedIndexNoticeOnScheduleForUpdateWithoutChanges()
 
         // This commit should not raise an E_NOTICE
         $this->_unitOfWork->commit();
+
+        $this->addToAssertionCount(1);
```

Comment:
This test creates a `$user`: it is sufficient for us to check if `$user` is in the unit of work, and whether it's still scheduled for update.